### PR TITLE
added script for csv to tf records

### PR DESCRIPTION
**Pull request by mikemilic98**
Hey cam so I found the script and make the necessary adjustments.

### CODE REVIEW

File: scripts/csv_to_tfrecord/generate_tf_record.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        'image/object/bbox/ymax': dataset_util.float_list_feature(ymaxs),
+        'image/object/class/text': dataset_util.bytes_list_feature(classes_text),
+        'image/object/class/label': dataset_util.int64_list_feature(classes),
+    }))
+    return tf_example
+
+
+def main(_):
+    writer = tf.python_io.TFRecordWriter(FLAGS.output_path)
+    path = os.path.join(os.getcwd(), 'images')
```
Comment (newtoncn) [inline scripts/csv_to_tfrecord/generate_tf_record.py]: So it looks like the script is looking for a folder called images in the current working directory, which in our implementation won't be the structure we're looking for. Can we fix this to look in the data/train_images folder, or allow us to specify if we want to use train_images or test_images? Once Jodi's csv script is in we should be able to test that this script will look in the right folder

File: scripts/csv_to_tfrecord/generate_tf_record.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        tf_example = create_tf_example(group, path)
+        writer.write(tf_example.SerializeToString())
+
+    writer.close()
+    output_path = os.path.join(os.getcwd(), FLAGS.output_path)
+    print('Successfully created the TFRecords: {}'.format(output_path))
+
+
+if __name__ == '__main__':
+tf.app.run()
```
Comment (newtoncn) [inline scripts/csv_to_tfrecord/generate_tf_record.py]: When I run python3 generate_tf_records.py I get this error 
File "generate_tf_record.py", line 94
    tf.app.run()
     ^
IndentationError: expected an indented block
Which I'm guessing is complaining about this indent. Usually under the if __name__ == 'main': tag you just run the main() function, should try running the script once the csv script is integrated to test this out.

File: scripts/csv_to_tfrecord/generate_tf_record.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        None
+
+
+def split(df, group):
+    data = namedtuple('data', ['filename', 'object'])
+    gb = df.groupby(group)
+    return [data(filename, gb.get_group(x)) for filename, x in zip(gb.groups.keys(), gb.groups)]
+
+
+def create_tf_example(group, path):
```
Comment (newtoncn) [inline scripts/csv_to_tfrecord/generate_tf_record.py]: Is this to create a tf record example of to create the actual tf records file? I'm definitely down for creating an example tfrecords file for testing but if this is the actual tf records script then this should just be create_tf_records_images or something along those lines.

File: scripts/csv_to_tfrecord/generate_tf_record.py [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+from object_detection.utils import dataset_util
+from collections import namedtuple, OrderedDict
+
+flags = tf.app.flags
+flags.DEFINE_string('csv_input', '', 'Path to the CSV input')
+flags.DEFINE_string('output_path', '', 'Path to output TFRecord')
+FLAGS = flags.FLAGS
+
+
+# TO-DO replace this with label map
```
Comment (newtoncn) [inline scripts/csv_to_tfrecord/generate_tf_record.py]: Should replace any of these comments with descriptions of the function or remove the function if it is unused.Probably should add some quick function definitions for any of the other functions as well, could use javadoc specifications

**Review by newtoncn — COMMENTED** [review]
Just a couple of quick comments, also we should just test that the script compiles and looks in the proper directories for our repository. Thanks for adding the file description to the top! Great for formatting, now in 30 years when they look back at this script they'll know Mike Milic wrote this!

**Comment by newtoncn** [comment]
Also looks like we need to merge this branch into master to keep it up to date, any time there's new changes to master we should git merge origin/master after running git fetch just to make sure our branch is up to date with the newest master branch and any merge conflicts are addressed

File: scripts/csv_to_tfrecord/generate_tf_record.py (hunk 3 above)
Comment (mikemilic98) [reply scripts/csv_to_tfrecord/generate_tf_record.py]: Changed tf_example to tf_record_image and create_tf_example to create_tf_records_images

File: scripts/csv_to_tfrecord/generate_tf_record.py (hunk 2 above)
Comment (mikemilic98) [reply scripts/csv_to_tfrecord/generate_tf_record.py]: fixed the indent left as tf.app.run() for now

File: scripts/csv_to_tfrecord/generate_tf_record.py (hunk 4 above)
Comment (mikemilic98) [reply scripts/csv_to_tfrecord/generate_tf_record.py]: the function is used on line 60 inside the create_tf_records_images, I added a description

**Review by newtoncn — APPROVED** [review]

**Pull request merged**